R/psis_approximate_posterior.R

```diff
@@ -0,0 +1,46 @@
+#' Diagnostics for Laplace and ADVI approximations and Laplace-loo and ADVI-loo
+#'
+#' @param log_p The log-posterior (target) evaluated at samples from the proposal distribution (q).
+#' @param log_q The log-density (proposal) evaluated at samples from the proposal distribution (q).
```

Comment:
Can you clarify in the doc that `log_p` and `log_q` should be vectors (right?) and  the same length?

Reply:
Yes! Ill do that!